src/includes.ts

```diff
@@ -0,0 +1,6 @@
+export const includes = (array: Array<any>, searchElement: any): boolean => {
```

Comment:
```ts
<T>(array: Array<T>, searchElement: T): boolean =>
  array.findIndex((element) => element === searchElement) !== -1
```

No needs to return boolean yourself and the T is needed, because you could have an array of string and pass a number as second argument otherwise 😄

Reply:
Ok